<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>once_run</class>
 <widget class="QWidget" name="once_run">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>运行一次</string>
  </property>
  <widget class="QTableWidget" name="once_run_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1041</width>
     <height>400</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <row>
    <property name="text">
     <string>**</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>客户编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>间隔时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>到达时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>服务时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>服务开始时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>等待时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>服务结束时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>客户在系统花费时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>空闲时间</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="avg_lb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>平均等待时间：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="avg_txt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="sys_use_lb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>系统利用率：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="sys_use_txt">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="help_btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_btn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clr_cache_btn">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>440</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除缓存数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="once_run_btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>运行一次</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
